core/src/test/java/io/questdb/test/griffin/SqlParserTest.java

```diff
@@ -7985,6 +8107,7 @@ public void testUndefinedBindVariables() throws SqlException {
     }
 
     @Test
+    @Ignore
```

Comment:
please remove `@Ignore`

Reply:
Removed in https://github.com/questdb/questdb/pull/3644/commits/d8eb986b70b3cb5cb585705d1253e027a316681c